src/arr_add.py

```
@@ -0,0 +1,722 @@
+        if radarr_existing.get("status") == "exists":
+            detail = str(radarr_existing.get("detail") or "")
+            console.print(f"[red]Radarr add skipped before qBittorrent search: {detail}[/red]")
+            movie_key = _radarr_add_duplicate_key_from_movie(radarr_existing.get("movie"))
+            await _remember_radarr_add_key(duplicate_key or movie_key, seen_title_years, seen_key_file)
+            return True
+
+    console.print(f"[green]Radarr add: gathering tracker IDs for {os.path.basename(path)}[/green]")
+
+    try:
+        await Clients(config).get_pathed_torrents(path, meta)
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)